examples/ethereum/rosetta-cli-conf/devnet/config.json

```diff
@@ -0,0 +1,60 @@
+{
+  "network": {
+    "blockchain": "Ethereum",
+    "network": "Mainnet"
```

Comment:
why does network says `Mainnet` here. Is that the correct network id for devnet?

Reply:
that is just a string Rosetta use to find chainid + network id. We can change it but we need to add support code in Rosetta as well

Reply:
will try to add the support code when enable check:construction of ERC20